service/history/configs/config.go

```diff
@@ -332,7 +335,7 @@ func NewConfig(dc *dynamicconfig.Collection, numberOfShards int32, isAdvancedVis
 		ReplicationTaskProcessorStartWait:                      dc.GetDurationPropertyFilteredByShardID(dynamicconfig.ReplicationTaskProcessorStartWait, 5*time.Second),
 		ReplicationTaskProcessorStartWaitJitterCoefficient:     dc.GetFloat64PropertyFilteredByShardID(dynamicconfig.ReplicationTaskProcessorStartWaitJitterCoefficient, 0.9),
 		ReplicationTaskProcessorHostQPS:                        dc.GetFloat64Property(dynamicconfig.ReplicationTaskProcessorHostQPS, 1500),
-		ReplicationTaskProcessorShardQPS:                       dc.GetFloat64Property(dynamicconfig.ReplicationTaskProcessorShardQPS, 5),
+		ReplicationTaskProcessorShardQPS:                       dc.GetFloat64Property(dynamicconfig.ReplicationTaskProcessorShardQPS, 30),
```

Comment:
These per shard rate limits are quite dangerous as it heavily depends on cluster configuration and super hard to come up with a reasonable default.  We should try to convert all per shard config knobs to something which is more manageable like per host

Reply:
there is a per host configuration just one line above: `ReplicationTaskProcessorHostQPS`

Reply:
I believe the long term solution (isolation) is to use 2 level throttler, first throttled by shard, then by host